client/blocks/product-purchase-features-list/sell-online-paypal.jsx

```diff
@@ -10,25 +10,46 @@ import { localize } from 'i18n-calypso';
  */
 import PurchaseDetail from 'components/purchase-detail';
 import { localizeUrl } from 'lib/i18n-utils';
+import { isEnabled } from 'config';
 
 /**
  * Image dependencies
  */
 import paymentsImage from 'assets/images/illustrations/payments.svg';
 
 export default localize( ( { isJetpack, translate } ) => {
+	let supportDocLink;
+	if ( isEnabled( 'earn/rename-payment-blocks' ) ) {
+		supportDocLink = localizeUrl(
+			isJetpack
+				? 'https://jetpack.com/support/pay-with-paypal/'
+				: 'https://wordpress.com/support/pay-with-paypal/'
```

Comment:
Do we know if this page is supposed to be public yet? It is, which is weird when https://wordpress.com/support/wordpress-editor/blocks/simple-payments-block/ still exists (I guess the latter gets unpublished when we flip the switch on the former?).

Reply:
Ah true @kwight ! We can likely unpublish the new ones until we're closer to July 7th

Reply:
Good call, Kirk! I just switched them to draft.